packages/front-end/pages/settings/index.tsx

```diff
@@ -92,7 +92,7 @@ const GeneralSettingsPage = (): React.ReactElement => {
     defaultValues: {
       visualEditorEnabled: false,
       pastExperimentsMinLength: 6,
-      metricAnalysisDays: 90,
+      metricAnalysisDays: 90, // should we use DEFAULT_METRIC_ANALYSIS_DAYS here?
```

Comment:
Should we use the defaults from the shared library here?

I also see there is a DEFAULT_CONFIG, should we use that object instead?

Reply:
Those constants are only available on the back-end right now.  I think stick with hardcoded for now.  We can migrate all those constants to the new shared package later.